### ISSUE TITLE
WinAuth password issue after changing windows password

### ISSUE THREAD

**Issue by jbajde** [winauth/winauth#578]
I changed my windows password today and now I can't login into WinAuth. It says invalid password when it starts up (before entering anything). And my password doesn't work anymore.

Any help with fixing this would be much appreciated.

**Comment by winauth** [comment]
If you reset your password in window, rather than changing it, then you have also force changed the master password that breaks any app that uses it. Try putting the password back, open Winauth and turn off "encrypt only to be usable on this computer" in "Change Protection..."

**Comment by jbajde** [comment]
Unfortunately I can't change back to the old password as it is my company computer and it is impossible to reuse old passwords. Is there something else I can do?

**Comment by TheDarkerPhantom** [comment]
> Is there something else I can do?

No;  An Administrator can change your password for you.  However, even if they reset it to the same password, you might have to look to previous backups to restore your authenticators.
